curriculum/challenges/english/25-front-end-development/lecture-introduction-to-javascript-libraries-and-frameworks/674ba6876f7ada867135bb95.md

```diff
@@ -148,7 +143,7 @@ exportComponent function Cat() {
 
 ### --feedback--
 
-Review the middle of the video where this was discussed.
+Review the section where this was discussed.
```

Comment:
this could also be a little more specific. something like "review where exports was discussed". or something like that. this should be applied to all feedbacks